integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/BuildTest.java

```diff
@@ -284,10 +284,10 @@ private Boolean buildToFinish(String buildId, EnumSet<BuildStatus> isIn, EnumSet
             logger.debug("Gotten build with status: {}", build.getStatus());
             if (!build.getStatus().isFinal()) return false;
         } catch (RemoteResourceNotFoundException e) {
-            //expected
+            logger.error("Group Build with id:{} not present", buildId);
```

Comment:
The test should FAIL immediately in this case.

Reply:
sure